docs/usage.md

```diff
@@ -106,7 +94,7 @@ output and source paths are the same, some package managers may add missing data
 like checksums as dependency data is resolved. If this occurs from a clean git
 tree then the tree has the possibility to become dirty.*
 
-### Generate environment variables (general)
+### Generate environment variables
```

Comment:
...here, and several other " (general)" section renames that follow in this file that can be made into a preliminary patch making this commit an absolute breeze to review with `git show --collor-moved`

Reply:
Yep.